t/count.t

```diff
@@ -40,6 +40,6 @@ __DATA__
 --- request
 GET /t
 --- response_body
-size: 54
+size: 55
```

Comment:
This test is here for a reason. I think you should update the initial table size in the Lua code accordingly.

Reply:
my mistake dude, i start understanding that meaning now